scalding-parquet-scrooge/src/main/java/com/twitter/scalding/parquet/scrooge/ParquetScroogeOutputFormat.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2012 Twitter, Inc.
```

Comment:
Half the scalding repo has 2012. How about I do a follow up PR after this and update all of them?